Behavior/Resources/Localizable.xcstrings

```diff
@@ -134,6 +134,17 @@
         }
       }
     },
+    "EMMA_BRUNSKILL_BIO" : {
+      "extractionState" : "manual",
```

Comment:
Please fix the `manual` extraction state, Xcode automatically synthesizes the String catalog upon every build, meaning you don't need to declare the string keys yourself

Reply:
Thanks for the review Phillip! I've gone ahead and changed the extraction state. Feel free to let me know if there's anything else.